app-misc/qlcplus/qlcplus-4.14.1.ebuild

```
@@ -74,7 +68,7 @@ pkg_postinst() {
 }
 
 src_test() {
-	virtx emake check
+	virtx cmake_build check
```

Default `cmake_src_test` (using ctest) did not work?

```suggestion
	local -x QT_QPA_PLATFORM=offscreen
	cmake_build check
```
let's try without virtualx entirely.

---

works, thanks, was not aware about that option